bc/web/templates/homepage.html

```diff
@@ -1,7 +1,100 @@
 {% extends "base.html" %}
+{% load static %}
 {% block title %}Home{% endblock %}
 {% block content %}
-<h1 class="text-3xl py-4 font-semibold">Big Cases Bot</h1>
-<p class="text-base">Big Cases Bot knows about <em>{{ subscription_count }}</em> cases.</p>
+
+<section class="mt-16">
+    <div class="max-w-7xl mx-auto px-4 sm:px-6 md:px-10 flex gap-10 items-center justify-center md:justify-between">
+        <h1 class="font-montserrat max-w-md lg:max-w-full font-[800] text-4xl xs:text-2xl sm:text-4xl md:text-5xl lg:text-6xl xl:text-7xl">
+            Hi! I’m the bot that keeps you updated about court cases!
+        </h1>
+        <img src="{% static "png/full-body-left-hand.png" %}" class="hidden xs:block w-32 sm:w-52 md:w-72 lg:w-80 xl:w-[500px]">
+    </div>
+</section>
+
+<section class="mt-16 bg-black">
+    <div class="mx-auto px-4 sm:px-6 md:px-10 flex flex-col justify-center py-8 md:py-16 text-white">
+        <div class="grid grid-cols-1 sm:grid-cols-2 flex flex-col justify-center gap-10">
+            <div class="flex justify-center">
+                <img src="{% static "png/chat-app-logos-combined.png" %}" class="w-56 object-contain sm:w-64 md:w-72 lg:w-80 xl:w-[500px]">
+            </div>
+            <div class="flex w-full justify-center">
+                <div class="pt-6 max-w-xs md:max-w-sm lg:max-w-lg flex flex-col">
+                    <h1 class="font-montserrat font-[400] text-4xl sm:text-6xl md:text-6xl lg:text-7xl">
+                        I go where you go.
+                    </h1>
+                    <p class="pt-5 xl:pt-12 text-md md:text-xl lg:text-2xl xl:text-3xl font-light">
+                        Getting updates about cases is important, so I’ll keep you posted on Slack, Discord, MS Teams and more.
+                    </p>
+                    <button class="bg-amber-300 text-black mt-8 p-3 rounded-md md:w-2/3 lg:w-2/5">
+                        Join the Waitlist
+                    </button>
+                </div>
+            </div>
+        </div>
+        <div class="grid grid-cols-1 sm:grid-cols-3 gap-4 md:gap-10 lg:gap-x-52 pt-10">
+            <div class="flex flex-col pt-5">
+                <h4 class="font-bold lg:text-2xl">
+                    Big Cases
+                </h4>
+                <p class="text-sm pt-3 font-light lg:text-base">
+                    Originally hosted by USA Today, my Twitter and Mastodon bots post updates about the most important cases in the country.
+                    <br><br>
+                    <a href="#">Learn More</a>
+                </p>
+            </div>
```

Comment:
These are probably worth doing an include since they're repetitive and we'll be using them more as the homepage and site grow.

Reply:
got it! thanks for your comment